pkgs/tools/networking/nbd/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
   patches = [ ./dont-run-make-in-broken-systemd-subdir.patch ];
 
   buildInputs =
-    [ pkgconfig glib ]
+    [ pkgconfig glib which ]
```

Comment:
Can you add this to `nativeBuildInputs` instead, also `pkgconfig` belongs there.

Reply:
Gah! One day, I'll get something right the first time.

Reply:
You can also cargo-cult this knowledge for examples like `pkgconfig`. 
A rule of thumb is: Does the system (i.e x86_64) that builds the packages would I like to execute this tool during its build and have it therefor in needs the executable for its own cpu architecture or is it a dependencies that is needed for the target platform (aarch64), that the package is build for.